.env.example

```
@@ -1 +1,2 @@
-MONGO_URI=mongodb://localhost:27017/mentor-match
\ No newline at end of file
+MONGO_URI=mongodb://localhost:27017/mentor-match
+SESSION_SECRET=63tv/7v09CHCLfEFZY+GzljdLLkpepis5bXnyLv4JLI=
```

Don't store the session secret here :) We can just do this insteads
```suggestion
SESSION_SECRET=REPLACE_ME
```